Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/EPBrakeSystem.cs

```diff
@@ -97,20 +97,22 @@ public override void Update(float elapsedClockSeconds)
                 }
                 else
                 {
-                    demandedAutoCylPressurePSI = Math.Min(Math.Max(Car.Train.BrakeLine4, 0), 1) * MaxTripleValveCylPressurePSI;
+                    demandedAutoCylPressurePSI = Math.Min(Math.Max(Car.Train.BrakeLine4, 0), 1) * ServiceMaxCylPressurePSI;
```

Comment:
This breaks EP brakes if ORTSMaxServiceCylinderPressure is not defined in the .eng file, since ServiceMaxCylPressurePSI is initialized to 0.

Reply:
Seems I forgot I didn't initialize that. Added a check to set the service max cylinder pressure equal to the (actual) max cylinder pressure if the user doesn't specify otherwise